combat-trainer.lic

```diff
@@ -1497,13 +1497,16 @@ class SpellProcess
                      .select { |skill| @training_spells[skill]['day'] ? UserVars.sun['day'] : true }
     needs_training = game_state.sort_by_rate_then_rank(needs_training).first
     return unless needs_training
+    DRC.message "needs_training: #{needs_training}" if $debug_mode_ct
```

Comment:
Do these need to be `DRC.message`? Most debug statements in this file use `echo`

Reply:
Switching to DRC.message where I can. I like you can see the output in the console as it is visible (red).